src/main/java/uk/gov/hmcts/reform/demo/errorhandling/GlobalExceptionHandler.java

```diff
@@ -17,21 +16,20 @@
 public class GlobalExceptionHandler {
 
     /**
-     * Template exception handler, that handles a custom AccountNotFoundException,
-     * and returns a 404 in the standard format.
+     * Exception handler that handles validation errors to the controller,
+     * and returns a 400 bad request error code.
      * @param ex The exception that has been thrown.
-     * @param request The request made to the endpoint.
      * @return The error response, modelled using the ExceptionResponse object.
      */
-    @ExceptionHandler(AccountNotFoundException.class)
-    public ResponseEntity<ExceptionResponse> handleAccountNotFound(
-        AccountNotFoundException ex, WebRequest request) {
+    @ExceptionHandler(ConstraintViolationException.class)
+    public ResponseEntity<ExceptionResponse> handleMethodValidationError(
```

Comment:
can we call this method `handle` and then overload the method with the variations of the exceptions that need handling?